### ISSUE TITLE
RFE: Allow changing the Key Delimiter for accessing nested values

### ISSUE THREAD

**Issue by satyenr** [spf13/viper#170]
I am talking about this field of Viper struct:

```
// Delimiter that separates a list of keys
// used to access a nested value in one go
keyDelim string
```

This can be useful in the following circumstances:
1. The keys in the config file contain periods.
2. Config key is being generated by something that uses a different delimiter - can save an unnecessary strings.Replace().

Looking at the code, it should be a simple matter of adding a SetKeyDelim() method.

**Comment by oneassasin** [comment]
any news?

**Comment by sagikazarmark** [comment]
Fixed in #794
